components/Avatar/avatar.css

```diff
@@ -0,0 +1,79 @@
+@import "/styles.css";
+@import "/docs.css";
```

Comment:
you can add all these css files including avatar.css into one css file eg :- main.css and then import it and you can keep importing new files in main whenever you make a new component.

Reply:
okay I will do it.